package.json

```diff
@@ -4,6 +4,7 @@
   "type": "module",
   "scripts": {
     "build": "node ./app/adapters.js && react-router build",
+    "fulldev": "pm2 start ecosystem.config.cjs --no-daemon",
```

Comment:
This is great! But I think for this instance we should be running the `build` first and then the `start` command rather than `dev`

Reply:
Just to add - I think there should have been more direction on the ticket - my bad. But this is so someone like a researcher only has to run one command and isn't for us as developers